tutorials/roofit/rf408_RDataFrameToRooFit.C

```diff
@@ -67,8 +67,13 @@ void rf408_RDataFrameToRooFit()
   // - the constructor arguments for RooDataSet (they follow the same syntax as the usual RooDataSet constructors)
   // - the column names that RDataFrame should fill into the dataset
   //
+  // The returned object is not yet a RooDataSet, but an RResultPtr that will be
+  // lazy-evaluated once you call GetValue() on it. But we will only do this later
+  // after declaring all actions, so the event loop is only run once.
```

Comment:
Maybe let's be a bit more precise in the last sentence, something along those lines maybe?:  
"We will only evaluate the RResultPtr once all other RDataFrame related actions are declared. This way we trigger the event loop computation only once, which will improve the runtime significantly."